tools/format_python_tools.sh

```diff
@@ -9,10 +9,12 @@ cd "$SCRIPTPATH"
 
 source_venv "$VENV_DIR"
 echo "Installing requirements..."
-pip install -r requirements.txt
+pip3 install --upgrade pip
+pip3 install -r requirements.txt
 
 echo "Running Python format check..."
-python format_python_tools.py $1
+python3 format_python_tools.py $1
 
 echo "Running Python3 flake8 check..."
-flake8 . --exclude=*/venv/* --count --select=E901,E999,F821,F822,F823 --show-source --statistics
+python3 -m flake8 --version
+python3 -m flake8 . --exclude=*/venv/* --count --select=E9,F63,F72,F82 --show-source --statistics
```

Comment:
Are we preserving semantics here (roughly)?

Reply:
Yes.  This is a superset of the previous tests: E9 runs all E9xx tests and F82 runs all F82x tests.  We are forcing flake8 to run on Python 3 because it has a stricter syntax than legacy Python.